back/src/app.ts

```diff
@@ -95,6 +76,18 @@ io.on('connection', function (socket: Socket) {
   });
 });
 
-const server = http.listen(3000, function () {
-  console.log('listening on *:3000');
+const server = http.listen(3000, () => {
+  console.log(`Sockets listening on port: ${colors.green('3000')}`);
+  const database = envConstants.isApiMock ? 'Mock' : 'MongoDB';
+  console.log(`Using ${colors.cyan(database)} to storage sessions`);
+  if (!envConstants.isApiMock && envConstants.mongoUrl) {
+    connect(envConstants.mongoUrl, {
```

Comment:
Check example that we did on previous cloud class. You can see this file to create mongo connection:

https://github.com/Lemoncode/master-frontend-lemoncode/blob/master/07-cloud/03-docker/05-heroku-back/01-implementation/back/src/core/database/database.connection.ts